src/core/00_acr.tf

```diff
@@ -0,0 +1,4 @@
+data "azurerm_container_registry" "acr" {
```

Comment:
why so many datasource?
can we reference a terraform remote state instead? could it be more efficient?

Reply:
yes is correct, we can try with a second iteration of changes to avoid the datasources